lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,169 @@
+                    $currentTarget.attr('aria-expanded', !ariaExpandedState);
+                },
+
+                unEnroll: function (e) {
+                    var $currentTarget = this.$(e.currentTarget),
+                        track_info = $currentTarget.data('track-info'),
+                        courseId = $currentTarget.data('course-id'),
+                        courseNumber = $currentTarget.data('course-number'),
+                        courseName = $currentTarget.data('course-name'),
+                        certNameLang = $currentTarget.data('cert-name-long'),
+                        refundInfo = $currentTarget.data('refund-info');
+
+                    $('#track-info').html(_.sprintf(track_info, {
+                        course_number: _.sprintf(
+                            "<span id='unenroll_course_number'>%(courseNumber)s</span>", {courseNumber: courseNumber}
+                        ),
+                        course_name: _.sprintf(
+                            "<span id='unenroll_course_name'>%(courseName)s</span>", {courseName: courseName}
+                        ),
+                        cert_name_long: _.sprintf(
+                            "<span id='unenroll_cert_name'>%(certNameLang)s</span>", {certNameLang: certNameLang}
+                        )
+                    }, true));
+
+                    $('#refund-info').html(refundInfo);
+                    $('#unenroll_course_id').val(courseId);
+                },
+
+                emailSettings: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+
+                    $('#email_settings_course_id').val($currentTarget.data('course-id'));
+                    $('#email_settings_course_number').text($currentTarget.data('course-number'));
+
+                    if ($currentTarget.data('optout') === 'False') {
+                        $('#receive_emails').prop('checked', true);
+                    }
+                },
+
+                upgradeToVerified: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+                    $currentTarget.closest('.action-upgrade').data('user');
+                    $currentTarget.closest('.action-upgrade').data('course-id');
+                },
+
+                unRegisterBlockCourse: function (e) {
+                    var $currentTarget = this.$(e.currentTarget),
+                        courseId = $currentTarget.data('course-id'),
+                        courseNumber = $currentTarget.data('course-number'),
+                        courseName = $currentTarget.data('course-name');
+
+                    if (this.$('#block-course-msg').length) {
+                        $('.disable-look-unregister').click();
+                    }
+
+                    $('#track-info').html(_.sprintf(
+                        "<span id='unenroll_course_number'>%(courseNumber)s</span> " +
+                        "- <span id='unenroll_course_name'>%(courseName)s?</span>",
```

Comment:
These 2 lines both have double quotes for strings when they should have single. the id values for the elements should have the double quotes.